components/list/list-item-drag-drop-mixin.js

```diff
@@ -339,6 +339,8 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 					opacity: 0;
 				}
 				:host([selected]) d2l-list-item-drag-handle,
+				:host([current]) d2l-list-item-drag-handle,
+				:host([_focusing][_list-item-new-styles]) d2l-list-item-drag-handle,
```

Comment:
We are updating to universally display the drag handle on focus (confirming with Jeff on if this should be focus primary action vs general focus and if it's okay to show on the parent as well if on general focus)